.gitignore

```diff
@@ -16,6 +16,10 @@ media
 # in your Git repository. Update and uncomment the following line accordingly.
 staticfiles/
 
+
+#previous login page
+templates/registration/login14.html
```

Comment:
May I know why the file name has numbers in it? 

Can this be renamed to `login.html`?

Reply:
Hey @sanjaypradeep this is the previous login page i have added it into .gitignore

Reply:
There's no need to retain previous files. Kindly remove it entirely and do not add that to the .gitinore file. We should be able to revert back to previous commits if we need a file.